tests/entities/__snapshots__/assetHolders.test.ts.snap

```diff
@@ -7,7 +7,14 @@ Object {
     "nodes": Array [
       Object {
         "__typename": "AssetHolder",
-        "amount": "101",
+        "amount": "-1000000",
+        "did": "0x0000000000000000000000000000000000000000000000000000000000000000",
```

Comment:
Are you sure there really is a all 0 did, and this isn't the result of serializing something that was empty?

Reply:
I'll have to check once again.

Reply:
Block 132 had this
<img width="467" alt="Screenshot 2022-06-01 at 1 19 42 AM" src="https://user-images.githubusercontent.com/34747455/171272986-9e399eb3-4e55-4b83-90bb-12c071b4a3f1.png">

Reply:
Thanks for checking.

I think people can put anything that looks like a DID as the target and the chain doesn't check if it exists. So we might need to think about this a bit more.

If thats the case, we might need to create a place holder identity.

Slack thread when this caused me an issue: https://polymathnetwork.slack.com/archives/C01RLA867C5/p1649454506627789?thread_ts=1649347860.107079&cid=C01RLA867C5